### ISSUE TITLE
Error wggallery and TDMDownloads

### ISSUE THREAD

**Issue by GregMage** [ggoffy/wggallery#29]
If you install tdmdownloads, there is a conflict with wggallery.

http://localhost/XoopsCore25/htdocs/modules/wggallery/

Error: Error: Call to undefined function tdmdownloadsReturnBytes() in file /modules/tdmdownloads/xoops_version.php line 537

The file's include xoops_version.inc.php is a problem. 

I did not add this way to TDMDownloads, but apparently it is not a good thing.

**Comment by ggoffy** [comment]
hi Gregory

I have installed now tdmdownloads (current version from github) and I got no error.

BUT: I realized also sometimes such strange error (also with other modules) and normally it was a cache problem (cleaning cache and problem doesn't appear again). I think this could be related to the namespace usage. But I personally am not so good in XOOPS to find out the problem, because I do not understand, how xoops cache and namespaces working together. but maybe I am completely wrong
